pkg/scheduler/generic_scheduler.go

```diff
@@ -80,6 +87,24 @@ type genericScheduler struct {
 	nextStartNodeIndex       int
 }
 
+type pluginScore struct {
+	Name  string
+	Score int64
+}
+
+type pluginScoreList []pluginScore
+
+// NodeToPluginScores declares a map from node name to its PluginScoreList.
+type nodeToPluginScores map[string]pluginScoreList
```

Comment:
if this can be refactor as `framework.NodeScore`, as the struct is the same. or maybe we can rename the `NodeScore`

Reply:
I'd rather to rename the `NodeScore`, because the same name with different functions makes people confusing. 

However, I found too many references to the type `NodeScore`,  I'm not sure if it is worth to do such a big change for a small feature. (And I believe the name `NodeScore` makes the referencing code more readable. )

Reply:
the second paragraph convinced me👍

Reply:
thanks for your suggest : )